consoles/VNC.pm

```diff
@@ -782,25 +783,26 @@ sub send_update_request {
     my ($self) = @_;
 
     # after 2 seconds: send forced update
-    # after 3 seconds: force incremental update
     # after 4 seconds: turn off screen
     my $time_since_last_update = time - $self->_last_update_received;
 
     # if there were no updates, send a forced update request
     # to get a defined live sign. If that doesn't help, consider
     # the framebuffer outdated
     if ($self->_framebuffer) {
-        if ($time_since_last_update > 4) {
+        if ($self->_last_update_requested - $self->_last_update_received > 2) {
```

Comment:
Perhaps the constant 2 could be given a meaningful name, like `$grace_period` or similar.